src/chainparams.cpp

```diff
@@ -272,7 +272,7 @@ class CTestNetParams : public CChainParams {
         consensus.nPowMaxAdjustDown = 32; // 32% adjustment down
         consensus.nPowMaxAdjustUp = 16; // 16% adjustment up
         consensus.nPowTargetSpacing = 2.5 * 60;
-        consensus.fPowAllowMinDifficultyBlocks = true;
+        consensus.nPowAllowMinDifficultyBlocksFromHeight = 299187;
```

Comment:
I have not checked this height.

Reply:
299187 is what the testnet explorer is reporting as of now, so should be fine.

Reply:
Or we set this 299188 since that is what "From" implies.